### PR TITLE
PAYARA-2300 Added logging of servlet mappings for each module at boot

### PR DESCRIPTION
**Pull request by arjantijms**
PAYARA-2300

### CODE REVIEW

**Comment by arjantijms** [comment]
Jenkins quick test Payara 5 please

File: appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/ModuleDescriptor.java [hunk 1]
```diff
@@ -51,18 +54,27 @@
      * Returns the context root of the application if any
      * @return 
      */
-    public String getContextRoot();
+    String getContextRoot();
```
Comment (smillidge) [inline appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/ModuleDescriptor.java]: Why has the visibility changed as this is a breaking change? These classes are exposed via the public Payara Micro api and now these methods aren't visible.

Comment (arjantijms) [reply appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/ModuleDescriptor.java]: ModuleDescriptor is an interface. Methods in an interface are public by default, not package or anything else. The public access modifier is redundant to use for interface methods. I.e. removing or adding them makes no difference for the visibility.

**Comment by arjantijms** [comment]
Jenkins quick test Payara 5 please

**Comment by arjantijms** [comment]
Jenkins quick test Payara 5 please

**Pull request merged**